test/end_to_end/mockvnode_SUITE.erl

```diff
@@ -364,6 +365,85 @@ mock_vnode_loadexchangeandrebuild_tester(TupleBuckets, PType) ->
     {ExchangeState3a, 2} = testutil:start_receiver(),
     true = ExchangeState3a == clock_compare,
 
+    % Exchange with a one hour modified range - should see same differences
+    io:format("Repeat exchange with 1 hour modified range~n"),
+    MRH = convert_ts(os:timestamp()),
+    {ok, _P3mr1, GUID3mr1} = 
+        aae_exchange:start(full,
+                                [{exchange_vnodesendfun(VNNa), IndexNs}],
+                                [{exchange_vnodesendfun(VNPa), IndexNs}],
+                                NullRepairFun,
+                                ReturnFun,
+                                {filter, all, all, large, all,
+                                    {MRH - (60 * 60), MRH},
+                                    pre_hash},
+                                []),
+    io:format("Exchange id ~s~n", [GUID3mr1]),
+    {ExchangeState3mr1, 2} = testutil:start_receiver(),
```

Comment:
it's just a test, but you could match directly on clock_compare

Reply:
Accepted, but this just aligns with format of other similar tests.